proto/composerapis/astria/auctioneer/v1alpha1/optimistic_block.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package astria.auctioneer.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+import "astria/execution/v1alpha2/execution.proto";
+import "astria/sequencerblock/v1alpha1/block.proto";
+import "astria/primitive/v1/types.proto";
+
+// The "BaseBlock" is the information needed to simulate bundles on top of
+// a Sequencer block which may not have been committed yet.
+message BaseBlock {
+    // This is the block hash for the proposed block.
+    bytes sequencer_block_hash = 1;
+    // The hash of previous rollup block, which new block will be created on top of.
+    bytes prev_rollup_block_hash = 3;
+    // List of transactions to include in the new block.
+    repeated astria.sequencerblock.v1alpha1.RollupData transactions = 4;
+    // Timestamp to be used for new block.
+    google.protobuf.Timestamp timestamp = 5;
+}
+
+// Identifying metadata for blocks that have been successfully committed in the Sequencer.
+message BlockCommit {
+    // Height of the sequencer block that was committed.
+    uint64 height = 1;
+    // Hash of the sequencer block that was committed.
+    bytes block_hash = 2;
+}
+
+// The Sequencer will serve this to the aucitoneer
+service OptimisticBlockService {
+    // The Sequencer will stream the optimistic Sequencer block (filtered for the provided
+    // rollup id) to the Auctioneer.
+    rpc StreamOptimisticBlock(astria.primitive.v1.RollupId) returns (stream astria.sequencerblock.v1alpha1.FilteredSequencerBlock);
```

Comment:
nit: lets make a `message StreamOptimisticBlockRequest` which contains the rollup id instead of having it be the actual request object. I know this is what I did in my gist, but having the message wrapper provides flexibility for adding future parameters.

Similar on the response